benchmarks/randomized_benchmarking.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+import functools
+import numpy as np
+import cirq
+import cirq.experiments.qubit_characterizations as ceqc
```

Comment:
Prefer a descriptive name rather than ceqc

Reply:
"ceqc" --> cirq experiments qubit characterizations 

This convention is also used in pyle.

Reply:
We are changing the convention in pyle though.

Reply:
Can you suggest a name based on the new convention?

Reply:
Updated the code to directly import the two functions needed from the module. PTAL.